src/test/rustdoc/const-generics/auxiliary/extern_crate.rs

```diff
@@ -0,0 +1,18 @@
+// edition:2018
+#![feature(min_const_generics)]
+
+pub fn extern_fn<const N: usize>() -> impl Iterator<Item = [u8; N]> {
+    [[0; N]; N].iter().copied()
+}
```

Comment:
Can you also test that rustdoc doesn't care about the body of functions returning const generics? It would go in `rustdoc-ui/error-in-impl-trait` - exactly this function signature, but with `loop {}` or some other divergent body. See https://doc.rust-lang.org/nightly/rustdoc/advanced-features.html#interactions-between-platform-specific-docs for exactly what that's meant to test.